### PR TITLE
RWS interface improvement

### PR DESCRIPTION
**Pull request by mkatliar**

### CODE REVIEW

File: src/rws_interface.cpp [hunk 1]
```diff
@@ -985,6 +974,35 @@ std::string RWSInterface::getLogTextLatestEvent(const bool verbose)
...
+
+
+bool RWSInterface::getDigitalSignal(std::string const& signal_name)
+{
+  auto const value = getIOSignal(signal_name);
+
+  if (value != SystemConstants::IOSignals::HIGH && value != SystemConstants::IOSignals::LOW)
+    throw std::logic_error("Unexpected value \"" + value + "\" of a digital signal");
+    
+  return getIOSignal(signal_name) == SystemConstants::IOSignals::HIGH;
```
Comment (Wojcik98) [inline src/rws_interface.cpp:997]: Use `value` instead of `getIOSignal`

File: src/rws_state_machine_interface.cpp [hunk 2]
```diff
@@ -732,8 +732,8 @@ void RWSStateMachineInterface::toggleIOSignal(const std::string& iosignal)
   {
     for (int i = 0; i < max_number_of_attempts && !result; ++i)
     {
-      setIOSignal(iosignal, SystemConstants::IOSignals::LOW);
-      result = (getIOSignal(iosignal) == SystemConstants::IOSignals::LOW);
+      setDigitalSignal(iosignal, false);
+      result = (getDigitalSignal(iosignal) == false);
```
Comment (Wojcik98) [inline src/rws_state_machine_interface.cpp:736]: Maybe just `!getDigitalSignal()`?

File: src/rws_state_machine_interface.cpp [hunk 3]
```diff
@@ -742,8 +742,8 @@ void RWSStateMachineInterface::toggleIOSignal(const std::string& iosignal)
 
       for (int i = 0; i < max_number_of_attempts && !result; ++i)
       {
-        setIOSignal(iosignal, SystemConstants::IOSignals::HIGH);
-        result = (getIOSignal(iosignal) == SystemConstants::IOSignals::HIGH);
+        setDigitalSignal(iosignal, true);
+        result = (getDigitalSignal(iosignal) == true);
```
Comment (Wojcik98) [inline src/rws_state_machine_interface.cpp:746]: Similar as before, maybe just `getDigitalSignal()`?

File: src/rws_state_machine_interface.cpp (hunk 2 above)
Comment (mkatliar) [reply src/rws_state_machine_interface.cpp:736]: done

File: src/rws_interface.cpp (hunk 1 above)
Comment (mkatliar) [reply src/rws_interface.cpp:997]: Good point, thanks!
Fixed.

File: src/rws_state_machine_interface.cpp (hunk 3 above)
Comment (mkatliar) [reply src/rws_state_machine_interface.cpp]: done

**Pull request merged**